### PR TITLE
grantee cardinality for multiple grantee

### PR DESCRIPTION
**Pull request by Anton-Voronov1234**

### CODE REVIEW

File: input/fsh/treatment-consent-profile.fsh [hunk 1]
```diff
@@ -12,7 +12,7 @@ A profile of the Consent resource for modeling treatment consents
 * date 1..1
 * period 1..1
 * grantor 1..1
-* grantee 1..1
+* grantee 1..
```
Comment (mojitoj) [inline input/fsh/treatment-consent-profile.fsh:15]: why would we want to restrict this for only one? I think there are legitimate use cases where the permission is granted to more than one individual or a team or a role.

Comment (Anton-Voronov1234) [reply input/fsh/treatment-consent-profile.fsh:15]: Yes I agree. Sorry that was my intention. 1..* not 1..

Comment (Anton-Voronov1234) [reply input/fsh/treatment-consent-profile.fsh:15]: What is the syntax for 1 or more?

Comment (mojitoj) [reply input/fsh/treatment-consent-profile.fsh:15]: this is actually correct syntax for 1..* sorry I mixed up the left and right panel here :D

**Review by mojitoj — APPROVED** [review]

File: input/fsh/treatment-consent-profile.fsh (hunk 1 above)
Comment (Anton-Voronov1234) [reply input/fsh/treatment-consent-profile.fsh:15]: No issues! Yes I was fixing the fact that it was only 1 before.

**Pull request merged**